composer.json

```diff
@@ -23,9 +23,10 @@
       "php": "7.*",
       "ext-json": "*",
       "automattic/jetpack-connection": "1.36.0",
-      "automattic/jetpack-config": "1.6.0",
+      "automattic/jetpack-config": "1.6.x-dev",
       "automattic/jetpack-autoloader": "2.10.11",
       "automattic/jetpack-identity-crisis": "0.7.0",
+      "automattic/jetpack-sync": "1.29.x-dev",
```

Comment:
TODO: These dev versions are temporary to test this PR https://github.com/Automattic/jetpack/pull/22458

Reply:
Done [38cde8f](https://github.com/Automattic/woocommerce-payments/pull/3763/commits/38cde8f91548ce7996055d8b9fa1981e2146b2cf)